src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/LicenceCondition.java

```diff
@@ -55,6 +55,10 @@
     @OneToOne
     private LicenceConditionTypeMainCat licenceConditionTypeMainCat;
 
+    @JoinColumn(name = "LIC_COND_TYPE_SUB_CAT_ID")
+    @OneToOne
+    private StandardReference licenceConditionTypeSubCat;
```

Comment:
Interestingly this field is NULLABLE (but so is MainCat) - I have a feeling you might get an error or NullPointerException